Global.xcconfig

```diff
@@ -8,11 +8,18 @@ MS_SIM_ARCHS = x86_64 i386
 ARCHS = $(MS_ARM_ARCHS)
 VALID_ARCHS = $(MS_ARM_ARCHS)
 
+// This flag adds -fembed-bitcode-marker in the release configuration
```

Comment:
In addition to the comment, you might need to change two environment variables.
`VERSION_FILE` and `VERSION_FILENAME`. Eventually we are going to use `VERSION_FILENAME` but just not to break the build, I keep both variables. Please update before you merge to develop.